pkg/cmd/historical_logs.go

```diff
@@ -148,14 +149,23 @@ func FetchLogs(baseUrl string, logParameters *LogParameters, podname string, pod
 		return
 	}
 
+	out, err := exec.Command("bash", "-c", "oc whoami --show-token").Output()
+	if err != nil {
+		fmt.Println("Error", err)
+	}
+	out = out[:len(out)-1]
+	var bearer = "`Bearer " + string(out) + "`"
+	req.Header.Set("Authorization", bearer)
+
 	query := req.URL.Query()
-	query.Add("podname", podname)
-	query.Add("namespace", logParameters.Namespace)
-	query.Add("starttime", logParameters.StartTime)
-	query.Add("finishtime", logParameters.EndTime)
-	query.Add("maxlogs", strconv.Itoa(logParameters.Limit))
-	query.Add("level", logParameters.Level)
+	query.Add("/pod/:", podname)
+	query.Add("/namespace/:", logParameters.Namespace)
```

Comment:
@apeksha-khandelwal please remove these `:` in all calls. Also, I'm not sure if this one builds the URL correctly. Will the URL be in the correct order if one specifies only the pod name?